app/read_models/received_on_reports/projection.rb

```diff
@@ -0,0 +1,43 @@
+module ReceivedOnReports
+  class Projection
```

Comment:
Discussion - technically this isn't a "projection" could this be a Read Model or somthing else?

Reply:
Good point. I'll consider this when I do the next one and possibly rename this class after that.